README.md

```diff
@@ -37,22 +37,22 @@ Executables of DICAT have been created for most systems and can be found in the
 To open DICAT, simply double click on the executable.
 
 ### From the source code
-
-######Requirements: 
-
+
+######Requirements: 

```

Comment:
You need a space after the "#" to have GitHub format the header in the frontend.